### ISSUE TITLE
The BN layer disappears when U2NET is converted

### ISSUE THREAD

**Issue by zhenzhen1003** [PINTO0309/openvino2tensorflow#68]
#### 6. Issue Details
I have an U2net model in ONNX format, in which the BN layer is BatchNormlization, which is converted into the IR model of OpenVino and then fused into the convolution operation. When converted into TFLite through OpenVino2TensorFlow, only the convolution layer is left, and the BN layer is not transformed. Such model prediction is very inaccurate. What should I do? Thanks for your answer.

**Comment by zhenzhen1003** [comment]
This is the ONNX model
![image](https://user-images.githubusercontent.com/72842107/131238243-40a9362f-b992-4d6b-864e-76636ccf3bab.png)


This is the IR model
![image](https://user-images.githubusercontent.com/72842107/131238210-357872ed-5093-4a11-8420-b1174d43d4e0.png)

This is tensorflow model
![image](https://user-images.githubusercontent.com/72842107/131238258-9fc99425-b9e8-4d82-92b3-adad7cdf6645.png)

**Comment by PINTO0309** [comment]
During the optimization process, the BN is merged into Conv2D's Weights. This is not a particular problem. If you are concerned about this, you can calculate the weights by hand, although it is tedious. My tool does extreme optimization.

If you want to know the specifications of BN fusion, please see below.
https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_Model_Optimization_Techniques.html

**Comment by zhenzhen1003** [comment]
> During the optimization process, the BN is merged into **`Conv2D`** or **`GroupConvolution`** or **`DepthwiseConv2D`** Weights. This is not a particular problem. If you are concerned about this, you can calculate the weights by hand, although it is tedious. My tool does extreme optimization. Rather, the overall performance of the model has been improved.
> 
> If you want to know the specifications of BN fusion, please see below.
> https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_Model_Optimization_Techniques.html
> 
> If you have a problem with BatchNormalization disappearing, you can add a layer corresponding to Normalization to the .xml file yourself. BN fusion is performed by OpenVINO Model Optimizer, not by openvino2tensorflow.
> 
> You need to verify if the degradation in accuracy you describe is really due to BN fusion.
> How much difference is there in the output value right after ReLU?
> 
> I don't know how detailed your research actually was, so I could only give you this level of response at this point.
> 
> Have you read the following document? Have you not read them? I have no idea what you are reading and understanding, so I have to ask questions about everything.
> 
> * Converting a Model Using General Conversion Parameters
>   https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html
> 
> ```
>   --disable_fusing      Turn off fusing of linear operations to Convolution
>   --disable_resnet_optimization
>                         Turn off resnet optimization
>   --finegrain_fusing FINEGRAIN_FUSING
>                         Regex for layers/operations that won't be fused.
>                         Example: --finegrain_fusing Convolution1,.*Scale.*
>   --disable_gfusing     Turn off fusing of grouped convolutions
> ```
> 
> The following tools are very useful when debugging a split model.
> https://github.com/yas-sim/openvino-model-division-and-simple-custom-layer
> 
> I am very sorry if I have made you feel uncomfortable. But I always find that many people's questions are so fragmented that I don't always understand what they really mean. And for that research I spend hours on one question.

Thank you very much. Your answer is very useful to me, now I have understood the operation of BN layer fusion, thank you again

**Comment by PINTO0309** [comment]
Closed due to lack of progress.
